RELEASE.md

```diff
@@ -1,3 +1,285 @@
+# Release 2.0.0-rc0
```

Comment:
-rc1?

Reply:
Actually we dont specify rc0 or rc1. removed it.